inference-engine/src/transformations/src/transformations/op_conversions/simplify_ctc_greedy_decoder.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "itt.hpp"
+
+#include "transformations/op_conversions/simplify_ctc_greedy_decoder.hpp"
+
+#include <ngraph/opsets/opset6.hpp>
+#include <ngraph/rt_info.hpp>
+
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::SimplifyCTCGreedyDecoder, "SimplifyCTCGreedyDecoder", 0);
+
+ngraph::pass::SimplifyCTCGreedyDecoder::SimplifyCTCGreedyDecoder() {
```

Comment:
I would rather to rename this transformation because the current name does not reflect its purpose. Please use, for example, ConvertCTCGreedyDecoderSeqLenToCTCGreedyDecoder or ConvertCTCGreedyDecoderSeqLenToSeqMask

Reply:
@GlebKazantaev you suggested current transformation name. Please comment this name.

Reply:
`ConvertCTCGreedyDecoderSeqLenToCTCGreedyDecoder` - from my point of view is too long and unreadable. So that is why I suggested to use `SimplifyCTCGreedyDecoder`. Also we don't need to encode the purpose in transformation name as all transformation details must be added to its doxygen description so plugin developers can find transformation by operation name and read description.